packages/web-components/src/calendar/calendar.ts

```diff
@@ -0,0 +1,813 @@
+
+  /**
+   * The month-picker is visible
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: month-picker-visible
+   */
+  @attr({ attribute: 'month-picker-visible', mode: 'boolean' })
+  public monthPickerVisible?: boolean;
+
+  /**
+   * Show month picker on top of date picker when visible
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: month-picker-overlay
+   */
+  @attr({ attribute: 'month-picker-overlay', mode: 'boolean' })
+  public monthPickerOverlay?: boolean = false;
+
+  /**
+   * Show week numbers (1-53) before each week row
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: show-week-numbers
+   */
+  @attr({ attribute: 'show-week-numbers', mode: 'boolean' })
+  public showWeekNumbers?: boolean = false;
+
+  /**
+   * The type of filter on the calendar
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: calendar-filter
+   */
+  @attr
+  public calendarFilter?: CalendarFilter | undefined;
+
+  /**
+   * The days that are selectable with the filter work-week.
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: work-week
+   */
+  @attr
+  public workWeek?: DaysOfWeek[] | undefined;
+
+  /**
+   * The first day of the week for locale
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: first-day-of-week
+   */
+  @attr
+  public firstDayOfWeek?: DaysOfWeek | undefined;
+
+  /**
+   * Determines when the first week of the year should start
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: first-week-of-year
+   */
+  @attr
+  public firstWeekOfYear?: FirstWeekOfYear | undefined;
+
+  /**
+   * Show "Go to today" link at the link slot
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: show-slotted-link
+   */
+  @attr({ attribute: 'show-slotted-link', mode: 'boolean' })
+  public showSlottedLink?: boolean;
+
+  /**
+   * the month picker highlights the current month
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: highlight-current-month
+   */
+  @attr({ attribute: 'highlight-current-month', mode: 'boolean' })
+  public highlightCurrentMonth?: boolean;
+
+  /**
+   * the month picker highlights the selected month
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: highlight-selected-month
+   */
+  @attr({ attribute: 'highlight-selected-month', mode: 'boolean' })
+  public highlightSelectedMonth?: boolean;
+
+  /**
+   * the format in which weekdays are displayed (M W T)
+   */
+  @attr public weekdayFormat: WeekdayFormat = WeekdayFormat.narrow;
+
+  /**
+   * the year on the month picker
+   */
+  @attr public monthPickerYear: number = new Date().getFullYear();
+
+  /**
+   * the decade on the year picker
+   */
+  @attr public yearPickerDecade: number = this.monthPickerYear - (this.monthPickerYear % 10);
+
+  /**
+   * whether the year picker is open
+   */
+  @attr public yearPickerOpen: boolean = false;
+
+  /**
+   * keeps track of the current focused and active date on the day grid
+   */
+  protected navigatedDate: Date = new Date(`${this.year}/${this.month}/01`);
```

Comment:
Could you explain how you landed on the `YYYY/MM/DD` format here? I would have expected the `new Date(year, month, day)` overload here as the format you selected not [necessarily universally supported](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#date_time_string_format) (though it seems to work fine in all our supported browsers!)

Mostly I'm just curious, not really advocating for a change :)

Reply:
Thanks for this comment, Sean! I wasn't aware that this format isn't universally supported. I was originally using `MM-DD-YY` but that was causing bugs on Firefox, so I switched to  `YYYY/MM/DD`. Switching to `new Date(year, month, day)` seems like a better solution :)